ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -115,80 +115,83 @@
-            <row.headerCell>{{t 'resources.target.title'}}</row.headerCell>
-            <row.headerCell>{{t 'form.proxy.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.started.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.status.label'}}</row.headerCell>
-            <row.headerCell />
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each this.sorted as |model|}}
-            <body.row as |row|>
-              <row.headerCell class='nowrap'>
-                <Rose::Icon
-                  class={{concat 'session-status-' model.status}}
-                  @name='flight-icons/svg/entry-point-16'
-                  @size='medium'
-                />
+      <Hds::Table
+        @model={{this.sorted}}
+        @columns={{array
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'resources.target.title'))
+          (hash label=(t 'form.proxy.label'))
+          (hash label=(t 'form.started.label'))
+          (hash label=(t 'form.status.label'))
+          (hash label='')
+        }}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if (can 'read model' B.data)}}
+                {{#if B.data.isActive}}
+                  <Hds::Badge
+                    @text={{t 'resources.session.title'}}
+                    @icon='entry-point'
+                    @isIconOnly={{true}}
+                    @color='success'
+                    class='session-status-active'
+                  />
+                {{else}}
+                  <Hds::Badge
+                    @text={{t 'resources.session.title'}}
+                    @icon='entry-point'
+                    @isIconOnly={{true}}
+                  />
+                {{/if}}
                 <Copyable
-                  @text={{model.id}}
+                  @text={{B.data.id}}
                   @buttonText={{t 'actions.copy-to-clipboard'}}
                   @acknowledgeText={{t 'states.copied'}}
                 >
-                  <code>{{model.id}}</code>
+                  <code>{{B.data.id}}</code>
                 </Copyable>
-              </row.headerCell>
-              <row.cell>{{model.target.displayName}}</row.cell>
-              <row.cell>
-                {{!
-                  Two if's here because:
-                  1) Local proxy info is only relevant if the session is
-                     active or pending (aka isCancelable).  Once the session
-                     is canceled or in the process thereof, the local proxy
-                     may no longer be used.
-                  2) The session actually has proxy information associated
-                     with it.  It's possible for the user to start sessions
-                     from other devices (or even another tab), in which case we
-                     won't have the local proxy info available here.
-                }}
```

Comment:
Any reason this comment was deleted? Is it still relevant, if so it seemed pretty informational